packages/kuma-gui/src/app/control-planes/locales/en-us/index.yaml

```diff
@@ -15,6 +15,11 @@ main-overview:
       services: 'Services'
       data_plane_proxies: 'Data Plane Proxies'
       policies: Policies
+      descriptions:
+        mesh_services: Mesh*Services
```

Comment:
Yeah I think we do what you just suggested, just make it `MeshServices`, we can bring this up in our design sync also and if we come up with a better term we can circle back wdyt?